external/webserver/http_client.c

```diff
@@ -476,6 +477,30 @@ int http_parse_message(char *buf, int buf_len, int *method, char *url,
 						http_dispatch_url(client, req);
 						process_finish = true;
 						read_finish = true;
+
+						/* Process the Trailer */
+						header_field_value = http_keyvalue_list_find(params, "Trailer");
```

Comment:
how about to make this code to function?

Reply:
done